cli/commands/preview/update.ts

```diff
@@ -73,16 +74,26 @@ async function runCommand(
 	}
 }
 
-export const registerCommand: RegisterCommand = ( program ) => {
-	program
-		.command( 'update [folder]' )
-		.description(
-			__( 'Update preview site for the specified folder (defaults to current directory)' )
-		)
-		.requiredOption( '-h, --host <host>', __( 'Host of the preview site to update' ) )
-		.action( async ( siteFolder: string = process.cwd(), options ) => {
-			const parentOptions = program.opts();
-			const normalizedHost = normalizeHostname( options.host );
-			await runCommand( siteFolder, normalizedHost, parentOptions.outputFormat );
-		} );
+export const registerCommand = ( yargs: Argv< GlobalOptions > ) => {
+	return yargs.command( {
+		command: 'update [folder]',
+		describe: __( 'Update preview site for the specified folder (defaults to current directory)' ),
+		builder: ( yargs: Argv< GlobalOptions > ) => {
+			return yargs
+				.positional( 'folder', {
+					type: 'string',
+					default: process.cwd(),
+					description: __( 'The folder to update the preview from' ),
+				} )
+				.option( 'host', {
+					type: 'string',
```

Comment:
```suggestion
					alias: 'h',
					type: 'string',
```
I'm not sure what I think the best approach is here, but I wanted to raise the question. `-h` is what we had before, but I guess you could argue that it could be confused with `--help`. WDYT, @bcotrim?

Reply:
I added the alias for `H`.
I think it makes it less likely to be confused with `--help`
What do you think?

Reply:
Good call 👍